README.md

```diff
@@ -25,98 +50,75 @@ The following helpers are supported and automatically selected, if present, in t
 |skip_pgp_check |yes, **no**                                                  |Only valid with makepkg. Skip PGP signatures verification of source file, useful when installing packages without GnuPG properly configured.|
 |ignore_arch    |yes, **no**                                                  |Only valid with makepkg. Ignore a missing or incomplete arch field, useful when the PKGBUILD does not have the arch=('yourarch') field.|
 
-### Note
+#### Note
 * Either *name* or *upgrade* is required, both cannot be used together.
 * In the *use*=*auto* mode, makepkg is used as a fallback if no known helper is found.
 
-## Installing
-### AUR package
-The [ansible-aur-git](https://aur.archlinux.org/packages/ansible-aur-git) package is available in the AUR.
-
-Note: The module is installed in `/usr/share/ansible/plugins/modules` which is one of the default module library paths.
-
-### Manual installation
-Just clone the *ansible-aur* repository into your user custom-module directory:
```

Comment:
why is the manual installation in the user custom-module directory not supported any longer?

Reply:
Technically, it still works. However, when the `aur` module is installed as a local custom module, its FQCN is `ansible.legacy.aur`. It is not required to use this FQCN (you can use the short name `aur`) in standalone roles and playbooks, but it is required in roles inside collections.

If you prefer to copy `aur.py` into the user custom module directory for development instead of installing the collection itself (see [this section](https://github.com/gotmax23/ansible-aur/tree/move_to_collection/#install-locally-for-development) of the new README), that works. For end users, I think it makes most sense to focus on one package (a collection) that is versioned and consistent. As I said, we could ship the same collection on both the AUR and on Ansible Galaxy. If you really want, I could add this back in along with the explanation above, but I worry that will confuse end users.

Please see [my earlier comment](https://github.com/gotmax23/ansible-aur/tree/move_to_collection/#install-locally-for-development) where I also touch on this issue.

Thanks,
Maxwell

Reply:
The local development of modules as presented in [ansible developing locally](https://docs.ansible.com/ansible/latest/dev_guide/developing_locally.html) is not considered legacy from what I read and is a simple and popular solution. I see no reason not to present how it can still be simply installed without the galaxy's bells and whistles. Can you keep a description of a local installation in your commit?

Reply:
I can add this local installation section if you are not available anymore.

Reply:
I apologize for not responding earlier; I've been busy. I should be able to work on this later today,

Reply:
I added instructions to install the `aur` module as a local custom module, as you asked. However, I don't recommend including this change. This change adds extra complexity and has the potential to confuse users, without any real benefit. Why can't we keep it simple and focus on one packaging method?

I also added a section about installing the collection through the `ansible-collection-kewlfft-aur` package on the AUR. I already created a PKGBUILD for this new package. I will push it to the AUR as soon as you merge this PR and create a new release.